scripts/utils/yarn.ts

```diff
@@ -20,7 +20,11 @@ export const addPackageResolutions = async ({ cwd, dryRun }: YarnOptions) => {
 
   const packageJsonPath = path.join(cwd, 'package.json');
   const packageJson = await readJSON(packageJsonPath);
-  packageJson.resolutions = { ...storybookVersions, 'enhanced-resolve': '~5.10.0' };
+  packageJson.resolutions = {
+    ...storybookVersions,
+    'enhanced-resolve': '~5.10.0', // TODO, remove this
+    playwright: '1.30.0', // this is for our CI test, ensure we use the same version as docker image
```

Comment:
could you mention that it should match circle config and code/package.json?

Reply:
sure